tests/conftest.py

```diff
@@ -964,6 +966,11 @@ def create_or_get_fanout(fanout_hosts, fanout_name, dut_host) -> FanoutHost | No
             fanout = fanout_hosts[fanout_name]
             if dut_host not in fanout.dut_hostnames:
                 fanout.dut_hostnames.append(dut_host)
+
+            # Ensure is_console_switch flag is consistent with parameter
+            if isinstance(fanout.host, SonicHost):
+                fanout.host.is_console_switch = is_console_switch
```

Comment:
Using `|=` instead of `=` could better? Then we can ensure it's `True` as long as it's ever set to `True`.